parameters/hardware/sensors/torso_laser_filters_parameters.yaml

```diff
@@ -12,6 +12,16 @@ scan_filter_chain:
         type: filters/MultiChannelMedianFilterFloat
         params:
           number_of_observations: 3
+- type: laser_filters/LaserScanBoxFilter
+  name: laser_box_filter
```

Comment:
@reinzor @MatthijsBurgh @MaxBaeten do you know if this is correct? (git blame says Rokus but he has nothing to do with it)

Reply:
Same